### ISSUE TITLE
Migrate to codeberg

### ISSUE THREAD

**Issue by PinguDEV-original** [datenanfragen/data#2712]
I know its not easy, but Github is by microsoft (versus Codeberg, which is hosted by a german non-profit organization) so it would be really cool to see a migration over to codeberg.

**Comment by zner0L** [comment]
I agree we should move to Codeberg, or similar, eventually and have brought this up in board meetings before. However, right now we are cautious to switch. For one, especially in the data repository we do get some small contributions from users who are not that deep in the FOSS community, but have a GitHub account. We think we might loose these contributors in the move.
More importantly, this repository in particular makes heavy use of GitHubs many CI integrations: We publish the database using Circle-CI and we test new PRs and add change suggestions to them using the GitHub API, which is very important in simplifying the review process. Last time we discussed this, forgejo had no mature support for CI, which is why moving wasn’t even an option. Still, I am following the development of CI in forgejo with interest in the hope we might be able to move at some point.

I suggest we keep this issue open for keeping track of the state of the discussion, as it is clearly of interest outside of board meetings.
